README.md

```diff
@@ -21,7 +21,15 @@ For more information please refer to the [documentation](https://krill.docs.nlne
 
 # Changelog
 
-## 0.9.5 RC8 'Have You considered these Upgrades?'
+## 0.9.5 RC9 'Have You considered these Upgrades?'
+
+NOTE: We found an issue with a test under Rust 1.59. We believe this issue may be
```

Comment:
This could be misunderstood to refer to a test of Rust itself. Perhaps instead say "when running one of the Krill tests under Rust 1.59" ?

Also "it was present in Krill 0.9.1" - does that refer to the issue in Tokio or the test in Krill? The latter I assume.

Other minor nits: "Rust" not "rust", and "Tokio" not "tokio".

Reply:
yeah, you're right I will think about rephrasing this..

The main point is that that it does not seem to be an issue affecting production tested code - although that code was compiled with an earlier Rust version.